src/components/App.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
```

Comment:
- Calculate needs to be imported to your app component 
![image](https://user-images.githubusercontent.com/46115725/80282266-26c83600-86d6-11ea-8b8a-1c684d84fc3b.png)